1-Draft/RFCNNNN-Simplify-Module-Manifests.md

```diff
@@ -342,6 +342,12 @@ On the plus side, this strategy allows for new metadata to be defined in manifes
 
 On the down side, it requires the management of additional files when working on manifests, which isn't necessarily simplifying manifest management.
 
+### Add a `-BackwardCompatible` parameter to `New-ModuleManifest`
+
+@iSazonov proposed we consider adding a `-BackwardCompatible` switch parameter to `New-ModuleManifest`. When that parameter is used, the manifest would be generated in a backward compatible format with new values being stored in `PrivateData`. Otherwise, it would generate the manifest in the latest format with new values stored at the root.
+
+Based on how parameters have been added in recent versions, that would work for now, but going forward how would this work? For example, if one or more new keys are added to PowerShell 7, and then others in 7.x or 8, how would you generate a manifest to target the appropriate downlevel version? For that reason, I don't believe this is a good option to move forward with.
```

Comment:
My thoughts was about compatibility versions < 5.1, `-BackwardCompatibleVer3`.
Recent version and newest ones (as I understand) will resolve this 
by themselves.

Reply:
I see, sorry that I misunderstood. So assuming that future versions of PowerShell (7+) allow for additional keys to be read from the root folder and perhaps not used without generating an error, that would work. I'll update that section.